content/en/network_monitoring/network_path/setup.md

```diff
@@ -31,8 +31,9 @@ Setting up Network Path involves configuring your Linux environment to monitor a
 
 ## Setup
 
-{{< tabs >}}
-{{% tab "Dynamic paths" %}}
+### Network Traffic Paths
+
+Configure network traffic paths to allow the Agent to automatically discover and monitor network routes based on actual network traffic, without requiring you to specify endpoints manually.
```

Comment:
```suggestion
Configure network traffic paths to allow the Agent to automatically discover and monitor network paths based on actual network traffic, without requiring you to specify endpoints manually.
```
Feel free to ignore if routes is correct here. I just think we should be consistent with the term.